src/main/java/org/springframework/samples/petclinic/repository/jdbc/JdbcPetRepositoryImpl.java

```diff
@@ -110,7 +110,7 @@ public void save(Pet pet) throws DataAccessException {
         } else {
             this.namedParameterJdbcTemplate.update(
                 "UPDATE pets SET name=:name, birth_date=:birth_date, type_id=:type_id, " +
-                    "owner_id=:owner_id WHERE id=:id",
+                    "owner_id=:owner_id, weight=: weight WHERE id=:id",
```

Comment:
I believe the whole context fails to start because of this accidental space.
Should be:
```suggestion
                    "owner_id=:owner_id, weight=:weight WHERE id=:id",
```

Reply:
I tested with a curl as you suggested and It worked for the listed endpoints.
It didn't fail but I have fixed the space issue now

Reply:
When I tried to run your version with the space – the whole service didn't even start. I think that's how it's supposed to be, so maybe you missed something during tests, but in any case – doesn't matter now.

Reply:
Thats fine